Objects/longobject.c

```diff
@@ -5278,8 +5280,20 @@ int_from_bytes_impl(PyTypeObject *type, PyObject *bytes_obj,
     return long_obj;
 }
 
+static PyObject *
+long_long_meth(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+    return long_long(self);
+}
+
+static PyObject *
+long_long_getter(PyObject *self, void *Py_UNUSED(ignored))
+{
+    return long_long(self);
+}
```

Comment:
What is the purpose of long_long_getter? Why not using long_long_meth?

Reply:
Avoid using the cast to `getter`.